src/engine/sys/sdl_input.cpp

```diff
@@ -1113,6 +1113,24 @@ static void IN_ProcessEvents( bool dropInput )
 				{
 					std::string text = e.text.text;
 
+					/* We can get a console key as a text event after we've opened the console from a key up/down event,
+					*  in a separate execution of IN_ProcessEvents()
+					*  The key character can also be anywhere in the text
+					*  This might be an SDL bug */
+					text.erase(
```

Comment:
This would break the feature where you can hold ALT to type a character that would otherwise be a console key (works on some Linux environments)

Reply:
That is where the ALT exemption is initially applied. But IIUC this new third level of console key filtering has no condition to check that, so such characters would be filtered out.